### ISSUE TITLE
Devour Magic

### ISSUE THREAD

**Issue by Technopig1992** [AzerothWarsLR/WarcraftLegacies#3090]
Blizzard recently changed Devour Magic to not work on Anti Magic Shield. This creates a unique issue for Scourge in Legacies as they do not have access to any other form of Dispel magic.

Should we make a custom Devour Magic to circumvent this or put regular AOE dispel on a unit.

**Comment by YakaryBovine** [comment]
@Technopig1992 Yeah good call logging this. Do you think it can wait until a `dev` release or is it more on the urgent side?

**Comment by Technopig1992** [comment]
> @Technopig1992 Yeah good call logging this. Do you think it can wait until a `dev` release or is it more on the urgent side?

I would of thought it to be more urgent but there hasnt been much noise around it (perhaps they think we cant fix it). 

I suppose it would depend on rough eta of the dev release (not exactly an easy one to answer)

**Comment by YakaryBovine** [comment]
Easily months for the Old God patch.
Ok, sounds like this is worth doing sooner rather than later so we'll do it into `main`. Note this is breaking the guideline as the fix is not going to be simple. But, high priority enough.

**Comment by Technopig1992** [comment]
Okay cool sounds good. So we will be going the route of having a custom version of Devour Magic that works on AMS, sounds ideal to me.

**Comment by YakaryBovine** [comment]
On further investigation, fixing this is going to be somewhere in between extremely difficult and outright impossible.
We've decided to just accept the new behaviour for now. I'm updating the ticket to say the tooltip needs to be updated to reflect the new behaviour.

**Comment by linaz2009** [comment]
give scourge/fel horde/warsong normal dispel too?
